schema/boundingVolume.schema.json

```diff
@@ -1,6 +1,7 @@
 {
-    "$schema" : "http://json-schema.org/draft-03/schema",
-    "title" : "boundingVolume",
+    "$schema" : "http://json-schema.org/draft-04/schema",
```

Comment:
In JSON schema v4, is there a way to say that only one of the three properties below can be present?  I changed this in #123.